.github/workflows/tests.yml

```
@@ -37,6 +37,24 @@ jobs:
         # the lack of a variable is _always_ false-y, and the defaults lack all
         # the special cases.
         include:
+          # Test with the version 2 of numpy comming soon.
+          - case-name: numpy2
+            os: ubuntu-latest
+            python-version: "3.12"
+            scipy-requirement: ""
+            numpy-requirement: "==2.0.0rc1"
```

```suggestion
            numpy-requirement: "==2.0.0rc2"
```
There is a new release candidate available